app/models/departure_fetcher.rb

```diff
@@ -1,16 +1,21 @@
+require 'duration'
+
 class DepartureFetcher
   include ActiveModel::SerializerSupport
 
   def initialize(params)
     @time = params[:time] ? Time.zone.parse(params[:time]) : Time.current
     @stop_id = params[:stop_id]
+    @time_limit = params.fetch(:time_limit, 10)
```

Comment:
Currently no one is passing that param, I just put it in there to make the class a bit less hard-cody. But yeah, I can add a to_i.